techniques/failures/F23.html

```diff
@@ -1,40 +1,60 @@
+   <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"></link>
+</head>
+<body>
+   <h1>Failure of 1.4.2 due to playing a sound longer than 3 seconds where there is no mechanism to turn it off</h1>
+   <section class="meta">
+      <p class="id">ID: F23</p><p class="technology">Technology: failures</p><p class="type">Type: Failure</p>
+   </section>
+   <section id="applicability">
+      <h2>When to Use</h2>
       <p> Applies to all technologies except those for voice interaction. </p>
-   </section><section id="description"><h2>Description</h2>
-      <p> This describes a failure condition for Success Criteria involving sound. If sound does not
-                        turn off automatically within 3 seconds and there is no way to turn the
-                        sound off then Success Criterion 1.4.2 would not be met. Sounds that play over 3 seconds
-                        when there is no mechanism to turn off the sound included in the content
-                        would fall within this failure condition. </p>
-   </section><section id="examples"><h2>Examples</h2>
+   </section>
+   <section id="description">
+      <h2>Description</h2>
+      <p>This describes a failure condition for Success Criteria involving sound. If sound does not
+         turn off automatically within 3 seconds and there is no way to turn the
+         sound off, independently from the overall system volume level, then Success Criterion 1.4.2 would not be met.
+         The sound would fall within this failure condition. </p>
+   </section>
+   <section id="examples">
+      <h2>Examples</h2>
       <section class="example">
-         
-            <ul>
-               <li> A site that plays continuous background music </li>
-            </ul>
-         
+         <ul>
+            <li>A site that plays continuous background music </li>
+         </ul>
       </section>
       <section class="example">
-         
-            <ul>
-               <li> A site with a narrator that lasts more than 3 seconds before
-                  	stopping, and there is no mechanism to stop it. </li>
-            </ul>
-         
+         <ul>
+            <li>A site with a narrator that lasts more than 3 seconds before
+               stopping, and there is no mechanism to stop it. </li>
+         </ul>
       </section>
-   </section><section id="tests"><h2>Tests</h2>
+   </section
+   ><section id="tests">
+      <h2>Tests</h2>
       <section class="procedure"><h3>Procedure</h3>
          <ol>
-            <li> Check that there is a way in a Web page to turn off any sound that
-                                    plays automatically for more than three seconds.</li>
+            <li>Check that there is a mechanism to turn off any sound that
+               plays automatically for more than three seconds, independently from the overall system volume level.
```

Comment:
So if the volume is the same as the system volume does the length of an auto-played audio file matter? I would hope that anything automatically played you could shut off after 3 minutes (regardless of the volume level).

Reply:
i think you're missing the point here - the "independently..." refers to the mechanism to turn off any sound. i.e the mechanism isn't "just turn off the audio of your entire system ... there ... now you satisfy the SC".

however, if this is confusing (that the "independently" refers to the mechanism), can be turned around to something like

`Check that there is a mechanism, independent from the overall system volume level, to turn off any sound that plays automatically for more than three seconds`